clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -70,41 +68,29 @@ object ElasticsearchIndex extends Elastic4sAutoDerivation {
   lazy val dateTimeFormatter: DateTimeFormatter =
     DateTimeFormatter.ofPattern("yyyy/MM/dd")
 
-  /**
-    * Creates an index using shapeless and reflection.
-    *
-    * @param fieldMapper The object defining how the Scala types of the fields in `Document` should be translated
-    *                    into Elasticsearch field types.
-    * @tparam Document The type of the Elasticsearch document, with a context bound also specifying that both an
-    *                  `implicit ctag: ClassTag[Document]` exists, plus an
-    *                  `implicit fieldMapper: FieldMapper[Document]` exists.
-    *                  https://www.scala-lang.org/files/archive/spec/2.12/07-implicits.html#context-bounds-and-view-bounds
-    * @return The index.
-    */
-  private[elasticsearch] def apply[
-    Document <: ClioDocument: ClassTag: FieldMapper: Indexable: HitReader
-  ](fieldMapper: ElasticsearchFieldMapper): ElasticsearchIndex[Document] = {
-    val esName =
-      ElasticsearchUtil
-        .toElasticsearchName(classTag[Document].runtimeClass.getSimpleName)
-        .stripPrefix("document_")
-
-    // We started without a version suffix, so we keep it that way to avoid breaking things.
-    val versionSuffix = if (fieldMapper.value == 1) "" else s"_v${fieldMapper.value}"
-
-    new ElasticsearchIndex[Document](s"$esName$versionSuffix", fieldMapper)
-  }
+  private val wgsUbamEsName = "wgs-ubam"
+  private val gvcfEsName = "gvcf-v2"
+  private val wgsCramEsName = "wgs-cram-v2"
```

Comment:
Not much point in having these as `val`s, since they're only used in one spot each.